frontend/src/components/Register.vue

```diff
@@ -152,7 +163,7 @@ export default {
                   class="style-chooser"
                   v-model="filterCities"
                   :options="cityOptions"
-                  :placeholder="$t('city')"
+                  :placeholder="$t('City*')"
```

Comment:
"$t('city')"  
this syntax is used by the translation ... and injects the appropriate word from JSON files in the assets folder..so please do not rename these, as then it breaks the translation..any change can be made to the JSON language files, if necessary..